cmd/kube-controller-manager/app/core.go

```diff
@@ -403,3 +404,15 @@ func startPVCProtectionController(ctx ControllerContext) (bool, error) {
 	}
 	return false, nil
 }
+
+func startPVProtectionController(ctx ControllerContext) (bool, error) {
+	if utilfeature.DefaultFeatureGate.Enabled(features.StorageProtection) {
+		go pvprotection.NewPVProtectionController(
+			ctx.InformerFactory.Core().V1().PersistentVolumes(),
+			ctx.InformerFactory.Core().V1().PersistentVolumeClaims(),
+			ctx.ClientBuilder.ClientOrDie("pv-protection-controller"),
```

Comment:
I think you need to add a policy to `pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go`.

Reply:
yeah, will add when finished the implementation